akka-remote/src/main/scala/akka/remote/serialization/DaemonMsgCreateSerializer.scala

```diff
@@ -129,13 +132,44 @@ private[akka] class DaemonMsgCreateSerializer(val system: ExtendedActorSystem) e
       supervisor = deserializeActorRef(system, proto.getSupervisor))
   }
 
-  protected def serialize(any: Any): ByteString = ByteString.copyFrom(serialization.serialize(any.asInstanceOf[AnyRef]).get)
+  private def oldSerialize(any: Any): ByteString = ByteString.copyFrom(serialization.serialize(any.asInstanceOf[AnyRef]).get)
+
+  private def serialize(any: Any): (Int, Boolean, String, Array[Byte]) = {
+    val m = any.asInstanceOf[AnyRef]
+    val serializer = serialization.findSerializerFor(m)
+
+    // this trixery is to retain backwards wire compatibility while at the same time
+    // allowing for usage of string manifests when enable-additional-serialization-bindings = on
```

Comment:
it's not (only) about `enable-additional-serialization-bindings`, it's about supporting `SerializerWithStringManifest` which can also be user defined serializer